src/content/docs/fr/reference/publish-to-npm.mdx

```diff
@@ -271,28 +269,34 @@ Vous avez besoin d'aide pour construire votre intégration, ou vous voulez simpl
 
 :::tip
- Besoin d'écraser les informations que notre bibliothèque lit de NPM ? Pas de problème ! [Créez une Issue GitHub](https://github.com/withastro/astro.build/issues/new/choose) avec les informations mises à jour et nous allons nous assurer que les informations personnalisées `name`, `description`, ou `homepage` sont bien utilisées à la place.
+ Besoin d'écraser les informations que notre bibliothèque lit de NPM ? Pas de problème ! [Créez un ticket GitHub](https://github.com/withastro/astro.build/issues/new/choose) avec les informations mises à jour et nous allons nous assurer que les informations personnalisées `name`, `description`, ou `homepage` sont bien utilisées à la place.
 :::
 
-### Collections
+### Catégories
+
+En plus du mot-clé obligatoire `astro-component` ou `withastro`, des mots-clés spéciaux sont également utilisés pour organiser automatiquement les paquets. L'inclusion de l'un des mots-clés ci-dessous ajoutera votre intégration à la catégorie correspondante dans notre bibliothèque d'intégrations.
 
-En plus du mot-clé obligatoire `astro-component` ou `withastro`, des mots-clés spéciaux sont également utilisés pour organiser automatiquement les Paquets. Inclure n'importe quel mots-clés ci-dessous ajoutera votre intégration à notre bibliothèque d'intégrations.
+| Catégorie              | Mots-clés                                    |
+|--------------------    |----------------------------------------------|
+| Accessibilité          | `a11y`, `accessibility`                      |
+| Adapteurs              | `astro-adapter`                              |
+| Analyse de données     | `analytics`                                  |
+| CSS + UI               | `css`, `ui`, `icon`, `icons`, `renderer`     |
+| Frameworks             | `renderer`                                   |
+| Chargeurs de contenu   | `astro-loader`                               |
+| Images + Média         | `media`, `image`, `images`, `video`, `audio` |
+| Performance + SEO      | `performance`, `perf`, `seo`, `optimization` |
+| Barre d'outils Dev     | `devtools`, `dev-overlay`, `dev-toolbar`     |
+| Utilitaires            | `tooling`, `utils`, `utility`                |
 
-| Collection         | Mots-clés                                    |
-|--------------------|----------------------------------------------|
-| Accessibilité      | `a11y`, `accessibility`                      |
-| Adapteurs          | `astro-adapter`                              |
-| Analyse de données | `analytics`                                  |
-| CSS + UI           | `css`, `ui`, `icon`, `icons`, `renderer`     |
-| Frameworks         | `renderer`                                   |
-| Performance + SEO  | `performance`, `perf`, `seo`, `optimization` |
+Les paquets qui n'incluent aucun mot-clé correspondant à une catégorie seront affichés comme `Uncategorized`.
```

Comment:
I didn't translate `Uncategorized` because it's between backticks and the [Integrations library](https://astro.build/integrations) is not available in French.
So, I also wonder if it is relevant to translate the name of the categories in the table above... Leaving them translated can help those who are looking to associate the right keywords but it can also be confusing if they are looking for the exact name in the library... I don't know which is better, so I translated the categories I added to the table since the others were already translated.

Reply:
It works as it is, thank you for taking the time in updating it🫡